lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -11,6 +11,13 @@ const Js::OpCode LowererMD::MDExtend32Opcode = Js::OpCode::MOVSXD;
 
 extern const IRType RegTypes[RegNumCount];
 
+bool
+LowererMDArch::IsArgSaveRequired(Func *func) {
```

Comment:
>IsArgSaveRequired [](start = 15, length = 17)

Actually there is nothing machine dependent in this API. So you can consider moving this to Lower.cpp